packages/core/src/combo-box/ComboBox.tsx

```diff
@@ -248,7 +249,7 @@ export const ComboBox = forwardRef(function ComboBox<Item>(
 
         break;
       case "Tab":
-        if (!multiselect && activeState) {
+        if (tabToSelect && activeState) {
```

Comment:
this is a breaking change as the single select menu used to select on TAB and now it only will, if you provide this prop

Reply:
@mark-tate @origami-z  yes, now with this prop only we will be able to select the typed item on tab key press. the behaviour is kept common for both single/multiselect combobox for consistency.
Let me know if we want to only apply this for multiselect only.

Reply:
From meeting: let's try default to different values in single / multi select

Reply:
Added the below change to not break the existing behaviour >>>
the default value for prop will be:
- true for single select
- false for multiselect

Reply:
Makes sense. Let's add some test for both

Reply:
@origami-z have added test.